rmake.py

```diff
@@ -59,12 +56,36 @@ def parse_args():
     parser.add_argument(      '--codecoverage', required=False, default=False, action='store_true',
                         help='Code coverage build. Requires Debug (-g|--debug) or RelWithDebInfo mode (-k|--relwithdebinfo), (optional, default: False)')
 
+    parser.add_argument(      '--compiler', type=str, required=False, default='g++', dest='compiler',
+                        help='Spcify path to host compiler.')
+
+    parser.add_argument('--cuda', '--use-cuda', dest='use_cuda', required=False, default=False,
+                        help='[DEPRECATED] Build library for CUDA backend. Deprecated, use HIP_PLATFORM environment variable to override default which is determined by `hipconfig --platform`')
```

Comment:
Needs action=store_true maybe or is that default action?     See cuda failed maybe something else